src/Model/Suppression/BaseResponse.php

```diff
@@ -20,49 +20,29 @@
  */
 abstract class BaseResponse implements ApiResponse
 {
-    /**
-     * @var string
-     */
     private $address;
-
-    /**
-     * @var string
-     */
     private $message;
 
-    /**
-     * @param string $address
-     * @param string $message
-     */
-    private function __construct($address, $message)
+    private function __construct(string $address, string $message)
     {
         $this->address = $address;
```

Comment:
We can move this to ::create

Reply:
What do you mean? To have empty constructor and set the properties in create method?

Reply:
Yeah, correct